### PR TITLE
Update data enrichment and data wrapper

### PR DESCRIPTION
**Pull request by AlasDiablo**

### CODE REVIEW

File: tdm-be/src/controller/data-enrichment.ts [hunk 1]
```diff
@@ -62,28 +61,80 @@ const router = express.Router();
...
-            });
+const getEnrichments = async () => {
+    const enrichments = [];
+
+    for (const enrichment of dynamicConfig.getConfig().enrichments) {
+        const ws = await axios.get<Enrichment>(enrichment.url);
+        const paths = ws.data.paths;
+        const servers = ws.data.servers;
+        const tags = enrichment.tags.flatMap((tag) => tag.name);
+        const excluded = enrichment.tags.flatMap((tag) => tag.excluded).filter((tag) => !!tag) as unknown as string[];
```
Comment (parmentf) [inline tdm-be/src/controller/data-enrichment.ts:102]: **thought (non-blocking):** ‏`filter((tag) => !!tag)` equivalent to `filter(Boolean)`?

**Review by parmentf — APPROVED** [review]
It seems good to me.

File: tdm-be/src/controller/data-enrichment.ts (hunk 1 above)
Comment (AlasDiablo) [reply tdm-be/src/controller/data-enrichment.ts:102]: I think so, but I prefer to make the filter coparator explicit.

Comment (parmentf) [reply tdm-be/src/controller/data-enrichment.ts:102]: I don't find `!!tag` explicit.
An explicit way would be `(tag) => Boolean(tag)`, which is equivalent to `Boolean`.

**Pull request merged**